src/AdminAcceptanceTests.Actions/Utils/EmailServerDriver.cs

```diff
@@ -57,27 +57,45 @@ public static async Task ClearEmailAsync(string hostUrl, string id)
 
         private static HttpClient NewHttpClient()
         {
-            ServicePointManager.ServerCertificateValidationCallback = delegate { return true; };
-            return new HttpClient();
+            HttpClientHandler handler = new HttpClientHandler();
+            handler.ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator;
```

Comment:
This could be condensed
```
var handler = new HttpClientHandler 
{
    ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator
};
```